clients/cloud/java-springboot/README.adoc

```diff
@@ -0,0 +1,172 @@
+= Spring Boot with Java Example Confluent Cloud Client
+
+NOTE: Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) using the Spring Boot using Java API.
+
+== Prerequisites
+
+* Java 1.8 or higher to run the demo application
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) cluster
+* Initialize a properties file at `src/main/resources/application-ccloud.properties` with configuration to your Confluent Cloud cluster:
+
+[source,shell]
+.application-ccloud.properties
+----
+❯ cat src/main/resources/application-ccloud.properties
+
+# topic config
+io.confluent.developer.config.topic.name=test
+io.confluent.developer.config.topic.replicas=3
+io.confluent.developer.config.topic.partitions=6
+
+# common configs
+spring.kafka.properties.ssl.endpoint.identification.algorithm=https
+spring.kafka.properties.sasl.mechanism=PLAIN
+spring.kafka.properties.request.timeout.ms=20000
+spring.kafka.properties.bootstrap.servers=bootstrap-server.confluent.cloud:9092
+spring.kafka.properties.retry.backoff.ms=500
+spring.kafka.properties.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="<API_KEY>" password="<API_SECRET>";
+spring.kafka.properties.security.protocol=SASL_SSL
+
+# Cloud SR Config
```

Comment:
There should be two separate examples: 

1. One without Avro/SR
2. One with Avro/SR

Reply:
I would submit separate PR for JSON version of this example.